go.mod

```diff
@@ -1,15 +1,28 @@
 module github.com/cloudwego/hertz
 
-go 1.16
+go 1.20
```

Comment:
> em... this may be a problem - since we have plenty of projects running under 1.20

This is just a precaution, currently LTS supported by go are 1.18-1.20, this PR can be left open without merging, maybe we can keep this PR open without merging until necessary.

https://go.dev/doc/devel/release#policy

Reply:
One more word, there are still many users using low versions of go and we need to be responsible for them. 
For this reason we have not used `io` instead of `ioutil` and upgraded the `x/sys` version,

Reply:
Got it, can I close it?